sdks/python/apache_beam/testing/benchmarks/nexmark/models/nexmark_model.py

```diff
@@ -26,27 +26,25 @@
   - The bid on an item for auction (Bid).
 
 """
+import json
 
 
 class Person(object):
   "Author of an auction or a bid."
 
   def __init__(
-      self, id, name, email, credit_card, city, state, timestamp, extra=None):
+      self, id, name, email, credit_card, city, state, date_time, extra=None):
     self.id = id
     self.name = name
-    self.email = email  # key
-    self.credit_card = credit_card
+    self.emailAddress = email  # key
```

Comment:
python's style convention uses snake case for fields, so this is normally self.email_address, ditto for below

Reply:
I did that because
```python
json.dumps(self.__dict__)
``` 
outputs fieldnames that is the same with the name of fields so doing camel case sort of lines up with java version. I'll try to look for ways to change the names